src/IECoreHoudini/bindings/IECoreHoudiniModule.cpp

```diff
@@ -110,7 +110,20 @@ static void *extractNodeFromHOM( PyObject *o )
 	
 	return OPgetDirector()->findNode( homNode->path().c_str() );
 }
-#if MIN_HOU_VERSION(16, 5, 0)
+
+#if defined( IECOREHOUDINI_WITH_QT5 )
+
+// This little function makes the OpenGL context for Houdini's main window
+// current. This can be necessary when wanting to create your own OpenGL context
+// which shares resources (textures, vertex buffers etc) with Houdini's contexts.
+// See GafferUI/GLWidget.py for an example of the hideous abuse this allows.
+static void makeMainGLContextCurrent()
```

Comment:
Does this function fail to compile against a Qt 4 Houdini, or fail at runtime, or always work? Just wondering if we need the #ifdef or if we can just always provide this function and then leave it to Gaffer to choose.

Reply:
I'll check, not sure. If I recall, we switched away from this sort of approach in H14 because it started returning false (and causing weird draw errors), and because they exposed the `QGLWidget` directly.

Though from my recent testing in H16.0(qt4) the gaffer viewport is drawing all white anyways, so maybe the shared widget thing isn't really working as expected anymore anyways.

Strangely enough, they do have both `RE_Render::makeCurrent()` and `RE_Render::makeCurrentQt()` though the later causes H16.5(qt5) to crash.

Reply:
Addressed in 4c76829c5a4ee58df10e1d1473f68258656dc75d